<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- param -->
	<param name="num_cameras" type="int" value="1"/>
	<param name="weights_path" type="str" value="$(find dnn_attitude_estimation)/weights/mle.pth"/>
	<!-- MAIN -->
	<node pkg="dnn_attitude_estimation" name="combine_mle_prediction" type="combine_mle_prediction.py" output="screen">
		<remap from="/image_raw0" to="/decompressed_image"/>
	</node>
</launch>
